lib/infobox/node.js

```diff
@@ -1,239 +1,249 @@
-define(['sorttable', 'snabbdom', 'd3-interpolate', 'helper', 'utils/node'],
-  function (SortTable, V, d3Interpolate, helper, nodef) {
-    'use strict';
-    V = V.default;
-
-    function showStatImg(o, d) {
-      var subst = {
-        '{NODE_ID}': d.node_id,
-        '{NODE_NAME}': d.hostname.replace(/[^a-z0-9\-]/ig, '_'),
-        '{NODE_CUSTOM}': d.hostname.replace(config.node_custom, '_'),
-        '{TIME}': d.lastseen.format('DDMMYYYYHmmss'),
-        '{LOCALE}': _.locale()
-      };
-      return helper.showStat(V, o, subst);
+import V from "snabbdom/dist/snabbdom-patch";
+
+import { SortTable } from "../sorttable";
+import * as helper from "../utils/helper";
+import nodef from "../utils/node";
+
+function showStatImg(o, d) {
+  var subst = {
+    "{NODE_ID}": d.node_id,
+    "{NODE_NAME}": d.hostname.replace(/[^a-z0-9\-]/gi, "_"),
+    "{NODE_CUSTOM}": d.hostname.replace(config.node_custom, "_"),
+    "{TIME}": d.lastseen.format("DDMMYYYYHmmss"),
+    "{LOCALE}": _.locale(),
+  };
+  return helper.showStat(V, o, subst);
+}
+
+function showHwImg(o, d) {
+  if (!d.model) {
+    return null;
+  }
+  var subst = {
+    "{MODEL}": d.model,
+    "{NODE_NAME}": d.hostname,
+    "{MODEL_HASH}": d.model.split("").reduce(function (a, b) {
+      a = (a << 5) - a + b.charCodeAt(0);
+      return a & a;
+    }, 0),
+    "{MODEL_NORMALIZED}": d.model
+      .toLowerCase()
+      .replace(/[^a-z0-9\.\-]+/gi, "-")
+      .replace(/^-+/, "")
+      .replace(/-+$/, ""),
+  };
+  return helper.showHwImg(V, o, subst);
+}
+
+export const Node = (el, d, linkScale, nodeDict) => {
```

Comment:
This produces an error as it can't be constructed using `new Node(...)`

```suggestion
export function Node(el, d, linkScale, nodeDict) {
```